dialog-el.html

```diff
@@ -296,7 +296,92 @@
       if (this.visible) this.close();
     };
 
-    proto.show = function() {
+    proto._resetPosition = function() {
+      this.style.top = 'initial';
+      this.style.right = 'initial';
+      this.style.bottom = 'initial';
+      this.style.left = 'initial';
+    }
+
+    proto.positionDialog = function(anchor) {
+      // If `anchor` is falsy OR the anchor and dialog don't share an
+      // offsetParent, early return
+      if (!anchor || anchor.offsetParent !== this.offsetParent) return;
```

Comment:
I have the same concern. I think it is not a safe assumption that the anchor element and the dialog will have the same offsetParent.